files/fr/web/performance/how_long_is_too_long/index.html

```diff
@@ -0,0 +1,28 @@
+
+<p>La première ressource récupérée après une demande est généralement un document HTML, qui appelle lui-même par la suite des ressources additionnelles. Comme noté dans la description du <a href="/fr/docs/Web/Performance/Critical_rendering_path">chemin critique de rendu (en anglais <i lang="en">critical rendering path</i>)</a>, lorsque la ressource est reçue alors les navigateurs commencent à traiter le HTML et affichent son rendu au fur et à mesure qu'il est reçu plutôt que d'attendre le chargement des ressources additionnelles.</p>
+
+<p>Oui, un temps de chargement d'une seconde est un bon objectif, mais c'est un objectif qui ne sera rempli que par quelques sites seulement. Les attentes diffèrent en réalité en fonction du contexte. L'affichage du texte «&nbsp;Bonjour tout le monde&nbsp;» devrait pouvoir s'afficher en quelques millisecondes sur un réseau d'entreprise, mais le téléchargement de la vidéo d'un chat rigolo sur un appareil de plus de 5 ans avec un réseau <i lang="en">Edge</i> dans le Nord de la Sibérie pourrait prendre plus de 20&nbsp;secondes sans choquer personne. D'une manière plus générale, si vous attendez trois ou quatre secondes avant de fournir le moindre contenu à la personne visitant votre site, vous perdrez des visiteurs potentiels qui ne reviendront peut-être jamais consulter votre site.</p>
+
+<p>En optimisant les performances de votre site, visez un temps de premier chargement du contenu ambitieux, comme par exemple 5 secondes sur un réseau mobile 3G, et 1,5 secondes sur un réseau Wifi, avec peut-être des objectifs de chargement de page encore plus ambitieux pour les chargements de page ultérieurs, en tirant parti des <i lang="en">service workers</i> et de la mise en cache. Il y a plusieurs moments différents pour initialiser le chargement de la page, le chargement des ressources additionnelles, la réponse à l'interaction de l'utilisateur ou de l'utilisatrice, et pour proposer des animations fluides :</p>
+
+<h3 id="idling_goal">Objectifs concernant le statut (en anglais <i lang="en">idling</i>)</h3>
+
+<p>Les navigateurs s'occupe des tâches les unes après les autres (même si des tâches de fond peuvent être prises en charge par les <i lang="en">web workers</i>). Cela signifie que l'interaction avec le visiteur, la peinture de la page et l'exécution des scripts ont lieu lors d'un processus unique. Si le processus est occupé à exécuter du code JavaScript complexe, le processus principal ne sera pas disponible pour réagir à la saisie du visiteur (par exemple le fait d'appuyer sur un bouton). Pour cette raison, le périmètre de l'exécution des scripts devrait être limité et divisé en plusieurs bouts de code pouvant chacun être exécuté en 50 millisecondes ou moins. Cela rend le processus disponible pour l'interaction avec la personne visitant le site.</p>
```

Comment:
"Les navigateurs s'occupent" instead of "Les navigateurs s'occupe"

Reply:
Thanks! Fixed in https://github.com/mdn/translated-content/commit/8a5dbb598d0e4d7f8ab9cb68ca6db216ea9eafb8